app/apps/server/bridges/livechat.ts

```diff
@@ -222,6 +223,14 @@ export class AppLivechatBridge extends LivechatBridge {
 		return LivechatDepartment.findEnabledWithAgents().map(boundConverter);
 	}
 
+	protected async _fetchLivechatRoomMessages(appId: string, roomId: string): Promise<Array<IMessage>> {
+		this.orch.debugLog(`The App ${ appId } is getting the transcript for livechat room ${ roomId }.`);
+		const messages = Livechat.getRoomMessages({ rid: roomId });
+		const messageConverter = this.orch.getConverters()?.get('messages');
+		const boundMessageConverter = messageConverter.convertMessage.bind(messageConverter);
```

Comment:
`messageConverter` may return undefined on line above, calling this would cause an error 👀

Reply:
Typescript should have caught that, right? 🤔

Reply:
That's true, TS should have caught it 👀 weird why it didn't

Reply:
Because of the optional chaining there TS said it was ok, but I'm throwing an error if this expression comes as `undefined` now